downstream/modules/platform/proc-controller-github-organization-setttings.adoc

```diff
@@ -1,37 +1,36 @@
 [id="proc-controller-github-organization-setttings"]
 
-= GitHub Organization settings
-
-When defining account authentication with either an organization or a team within an organization, you should use the specific organization and team settings.
-Account authentication can be limited by an organization and by a team within an organization.
+= Configuring GitHub organization authentication
 
+When defining account authentication with either an organization or a team within an organization, you should use the specific organization and team settings. Account authentication can be limited by an organization and by a team within an organization.
 You can also choose to permit all by specifying non-organization or non-team based settings.
+You can limit users who can login to the platform by limiting only those in an organization or on a team within an organization.
 
-You can limit users who can login to the controller by limiting only those in an organization or on a team within an organization.
-
-To set up social authentication for a GitHub Organization, you must obtain an OAuth2 key and secret for a web application. To do this, you must first register your organization-owned application at \https://github.com/organizations/<yourorg>/settings/applications.
+To set up social authentication for a GitHub organization, you must obtain an OAuth2 key and secret for a web application using the link:https://docs.github.com/en/apps/using-github-apps/installing-your-own-github-app[registering the new application with GitHub].
 
-To register the application, you must supply it with your Authorization callback URL, which is the *Callback URL* shown in the *Details* page.
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
+The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI. To register the application, you must supply it with your webpage URL, which is the Callback URL shown in the Authenticator details for your authenticator configuration. See xref:gw-display-auth-details[Displaying authenticator details] for instructions on accessing this information.
 
 .Procedure
-. From the navigation panel, select {MenuAEAdminSettings}.
-. On the *Settings* page, select *GitHub settings* from the list of *Authentication* options.
-. Select the *GitHub Organization* tab.
+. From the navigation panel, select {MenuAMAuthentication}.
```

Comment:
See the global update for this in above sections. (Need to add "Click **Create authentication**."

Reply:
Fixed.